File: app/src/main/java/com/ntech/fourtop/network/pojo/AppResponse.java
package com.ntech.fourtop.network.pojo;

import java.util.List;

public class AppResponse {

    private String dataString;
    private List<DataImage> dataImages;
    private DataToken dataToken;
    private DataVoting dataVoting;
    private DataProfile dataProfile;

    public AppResponse() {
    }

    public AppResponse(String dataString) {
        this.dataString = dataString;
    }

    public AppResponse(List<DataImage> dataImages) {
        this.dataImages = dataImages;
    }

    public AppResponse(DataToken dataToken) {
        this.dataToken = dataToken;
    }

    public AppResponse(DataVoting dataVoting) {
        this.dataVoting = dataVoting;
    }

    public AppResponse(DataProfile dataProfile) {
        this.dataProfile = dataProfile;
    }

    public String getDataString() {
        return dataString;
    }

    public List<DataImage> getDataImages() {
        return dataImages;
    }

    public DataToken getDataToken() {
        return dataToken;
    }

    public DataVoting getDataVoting() {
        return dataVoting;
    }

    public DataProfile getDataProfile() {
        return dataProfile;
    }
}

File: app/src/main/java/com/emikhalets/voteapp/network/pojo/AppResponse.kt
package com.emikhalets.voteapp.network.pojo

class AppResponse {
    var dataString: String? = null
        private set
    var dataImages: List<DataImage>? = null
        private set
    var dataToken: DataToken? = null
        private set
    var dataVoting: DataVoting? = null
        private set
    var dataProfile: DataProfile? = null
        private set

    constructor() {}
    constructor(dataString: String?) {
        this.dataString = dataString
    }

    constructor(dataImages: List<DataImage>?) {
        this.dataImages = dataImages
    }

    constructor(dataToken: DataToken?) {
        this.dataToken = dataToken
    }

    constructor(dataVoting: DataVoting?) {
        this.dataVoting = dataVoting
    }

    constructor(dataProfile: DataProfile?) {
        this.dataProfile = dataProfile
    }
}